test/statement/other.stmt

```diff
@@ -55,3 +55,25 @@ foo() async {
 foo() async {
   await i(1 + 2);
 }
+>>> Throw.
+throw   'error'
+;
+<<<
+throw 'error';
+>>> Throw with long string literal.
+throw new FormatException('This is a long exception message.');
+<<<
+throw new FormatException(
+  'This is a long exception message.',
+);
```

Comment:
Probably worth testing that it never splits after `throw`:

```
throw 'Some extremely long error message.';
```

Reply:
Yes! Added.